src/routes/Group/components/TicketView.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Table } from 'semantic-ui-react';
+import { Link } from 'react-router';
+
+const actionFormatting = {
+  CREATED: "Created",
+  PAGE_SENT: "Page Sent",
+  ACKNOWLEDGED: 'Acknowledged',
+  REJECTED: 'Rejected',
+  CLOSED: 'Closed'
+}
+
+class TicketView extends React.Component {
+
+  actionRow(action){
+    return (
+      <tr>
+      <td>{action.actionTaken}</td>
+      <td>{action.timestamp}</td>
+      <td>{action.userId}</td>
+      </tr>
+    )
+  }
+
+  userLink(user) {
+    return user ? <Link to={'/user/' + user._id}>{user.name}</Link> : <span>No one</span>;
+  };
+
+  ticketRow(ticket, index){
+    var title = (ticket.metadata.title) ? ticket.metadata.title : 'No name';
+    var rows = [];
+    var id = 'ticket_'+ticket._id+'_'+index;
+    if (index == 0) {
+      rows.push(<Table.Cell key={'title_'+id} rowSpan={ticket.actions.length}>{title}</Table.Cell>)
+    }
+    if (ticket.actions){
```

Comment:
need a space before the {. Make sure to lint files for things like this in general.